packages/mdc-select/mdc-select.scss

```diff
@@ -95,6 +98,8 @@
     transition:
       mdc-animation-exit(opacity, 125ms),
       mdc-animation-exit(transform, 125ms);
+    white-space: nowrap;
+    overflow: hidden;
```

Comment:
I did that originally, but - at least on iOS - the native select doesn't use it, so I elided it

<img width="613" alt="screen shot 2016-12-16 at 16 32 40" src="https://cloud.githubusercontent.com/assets/1185269/21279547/aec975b2-c3ae-11e6-84de-6407fbd48d34.png">